frontend/src/views/web/landing.html

```diff
@@ -80,26 +80,37 @@ <h4 class="grad-span">
                         <div class="align-center">
                             <h4 class="mons-med-font"><div>What is EvalAI all about?</h4>
                             <p><span class="text-dark-black w-500 mons-med-font">EvalAI is an open-source web platform for organizing and participating in competitions to push the state of the art on AI tasks.</span>
-                                <!-- <ul class="text-med-black">
-                                <li>
-                                    EvailAI details one performance on Artificial Intelligence challenges
-                                    <div class="big-line"></div>
-                                </li>
-                                <li>
-                                    EvailAI details two performance on Artificial Intelligence challenges
-                                    <div class="big-line"></div>
-                                </li>
-                                <li>
-                                    EvailAI details three performance on Artificial Intelligence challenges
-                                    <div class="big-line"></div>
-                                </li>
-                            </ul> -->
                             </p>
                         </div>
                     </div>
                 </div>
             </div>
-            <!-- footer -->
+        </section>
+        <!-- research labs section -->
+        <section class="ev-container ev-details text-med-black ev-super-light-bg" id="ev-scroll-down">
+            <div class="grad-container text-med-black ">
+                <div class="row">
+                    <div class="col s12">
+                        <div class="align-center">
+                            <h4 class="mons-med-font"><div>Who uses EvalAI?</h4>
+                            <p><span class="text-dark-black w-500 mons-med-font">Several Research Organizations have already started hosting their challenges on EvalAI. Some of them are</span>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="row center">
+                    <div class="col s1"></div>
+                    <div class="col s3 center">
```

Comment:
I would suggest to take away `width:100%` from here and add it to .scss by making an image class.
Also add `margin` property to the class.